optbinning/binning/binning_statistics.py

```diff
@@ -608,7 +608,7 @@ def build(self, show_digits=2, add_totals=True):
         return df
 
     def plot(self, metric="woe", add_special=True, add_missing=True,
-             style="bin", show_bin_labels=False, savefig=None, figsize=None):
+             style="bin", show_bin_labels=False, savefig=None, figsize=None, save_kwargs={}):
```

Comment:
A function definition with a mutable type ``{}`` is bad practice. Please change to ``save_kwargs=None`` and amend the code appropriately.

Reply:
Same comment for the definitions below. Thanks!